SimplenoteUITests/SimplenoteUISmokeTestsSettings.swift

```diff
@@ -0,0 +1,56 @@
+import XCTest
+
+class SimplenoteUISmokeTestsSettings: XCTestCase {
+
+    override class func setUp() {
+        app.launch()
+        getToAllNotes()
+        _ = attemptLogOut()
+        EmailLogin.open()
+        EmailLogin.logIn(email: testDataExistingEmail, password: testDataExistingPassword)
+        NoteList.waitForLoad()
+    }
+
+    override func setUpWithError() throws {
+        getToAllNotes()
+        NoteList.trashAllNotes()
+        Trash.empty()
+        NoteList.openAllNotes()
+    }
+
+    override class func tearDown() {
+        Settings.open()
+        Settings.condensedModeDisable()
+        Settings.close()
+    }
+
+    func testUsingCondensedNoteList() throws {
+        trackTest()
+        let note = NoteData(
+            name: "Condensed Mode Test",
+            content: "Condensed Mode Content",
+            tags: []
+        )
+
+        trackStep()
+        NoteList.openAllNotes()
+        NoteList.createNoteThenLeaveEditor(note)
+        NoteListAssert.noteContentIsShown(note)
+
+        trackStep()
+        Settings.open()
+        Settings.condensedModeEnable()
+        Settings.close()
+        NoteList.openAllNotes()
+        NoteListAssert.noteContentIsShownInSearch(noteName: note.name, expectedContent: "")
```

Comment:
This method name is misleading, because I initially created it to assert notes in search mode, but it appeared applicable for non-search mode too. I'm keeping it old in this PR to keep the noise down.